avidemux/qt4/i18n/avidemux_da.ts

```diff
@@ -5590,55 +4961,46 @@ p, li { white-space: pre-wrap; }
     <name>ffnvenc</name>
     <message>
         <location filename="../../../avidemux_plugins/ADM_videoEncoder/ffNvEnc/ADM_ffNvEnc.cpp" line="+192"/>
-        <location filename="../../../avidemux_plugins/ADM_videoEncoder/ffNvEncHEVC/ADM_ffNvEnc.cpp" line="+192"/>
```

Comment:
I'm very sorry for having to nitpick again, but source references should not get lost. Was your source tree really up to date? This looks for me like the `avidemux_plugins/ADM_videoEncoder/ffNvEncHEVC` folder was missing.